### ISSUE TITLE
Security Upgrades - Integerate DOMPurify

### ISSUE THREAD

**Issue by LostRhapsody** [LostRhapsody/slothtime#47]
Because we're accepting lots of user input, we should really clean it. This can be triggered using a simple event listener any time input is detected in the usual areas, time entry table, theme modal, and upcoming notes.

How necessary is this? Probably not very. But it's nice to have.

Soon, we'll be moving to Next.JS and have some server-side operations, including storing data. It would be prudent to clean any user input before we start sending it to the server.

https://github.com/cure53/DOMPurify
